controllers/MessagesController.js

```diff
@@ -18,15 +18,27 @@ const addMessage = async (req, res) => {
   }
 };
 
+const getOtherUser = async (req, res) => {
+  try {
+    const { userId2 } = req.params;
```

Comment:
I think I was the one who did this but while you're at it can you come up with a better name for these variables? Something to better distinguish which user we're talking about here and in `getDirectMessage` on like 34?

Would be also nice if we could have this same naming convention when the front end calls these endpoints in `DirectMessage.tsx`